### ISSUE TITLE
Small note insertion produces null pointer exception

### ISSUE THREAD

**Issue by rodolfo-medina** [Audiveris/audiveris#449]
On p.5 of attached score at measures 5 and 7 we have a small grace note.  When I insert it, Audiveris complains a Null pointer exception.  Why this?  Thanks

[schubert-sonata18d894-primo_tempo.pdf](https://github.com/Audiveris/audiveris/files/5703366/schubert-sonata18d894-primo_tempo.pdf)

**Comment by hbitteur** [comment]
Yes a NPE is raised because the grace stem has no shape defined.
I just fixed this in "development" branch.
Graces are now OK when inserted **as a whole** from the shape board.

But I discovered other problems, when building these items manually **piece by piece** (say the small head, then the stem, then small flags). This approach is needed when dealing with more than 1 flag, as in your input score.
I still need to work on this.
